### DISCUSSION TITLE
Instructions on how to remove/uninstall

### DISCUSSION THREAD

**Discussion by profetik-777** [danielmiessler/Fabric#378] [General]
Didn't see anything on documentation on how to remove/uninstall. I'm currently on linux (ubuntu).

Just remove the folder where everything lived?

**Comment by xssdoctor** [comment]
Pipx uninstall fabric

**Comment by computerworxdev** [comment]
how to uninstall on macos when installed using go?
how to clear the intsallation. i have problems and need to reinstall it

**Comment by ksylvan** [comment]
If you installed it using go, you can just remove the go binary in your `$HOME/go/bin`

On the Mac, you should be able to do, in a terminal:

```bash
rm ~/go/bin/fabric
```

Let me know that worked for you @computerworxdev

**Comment by computerworxdev** [comment]
thank you that removes the app not config
to completely remove the app i assume you have to delete config folder i have issue with 

```
sudo rm -rf ~/.config/fabric
and
rm ~/go/bin/fabric
```

**Comment by ksylvan** [comment]
Yes, that is correct. You don't really need the `sudo` though.

```bash
rm ~/go/bin/fabric && rm -rf ~/.config/fabric
```

That should do it.

**Comment by computerworxdev** [comment]
thank you I reinstall the fabric now and it works with a new ollama local model. (For some reason once set and you want to change to a different model fabric --setup does not work when making permanent change to model). Simply it won't let you in to choose a new model I think it might be a bug
